federation/pkg/kubefed/join.go

```diff
@@ -244,7 +244,7 @@ func (j *joinFederation) Run(f cmdutil.Factory, cmdOut io.Writer, config util.Ad
 			// 1. We want to create the secret in a specific namespace, which
 			//    is neither the "default" namespace nor the one specified
 			//    via the `--namespace` flag.
-			// 2. `SecretGeneratorV1` requires LiteralSources in a string-ified
```

Comment:
This is intentional too.  It means making something a string. I would leave this as it was before too.